x/issuance/types/genesis.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import "bytes"
+
+// GenesisState is the state that must be provided at genesis for the issuance module
+type GenesisState struct {
+	Params Params `json:"params" yaml:"params"`
```

Comment:
blocked address are not persisted on genesis state?

Reply:
They are - blocked addresses is a property of asset, which is a property of params.